src/main/java/com/launchdarkly/client/LDConfig.java

```diff
@@ -24,6 +24,7 @@
   private static final long DEFAULT_POLLING_INTERVAL_MILLIS = 1000L;
   private static final long DEFAULT_START_WAIT_MILLIS = 5000L;
   private static final int DEFAULT_SAMPLING_INTERVAL = 0;
+  private static final long DEFAULT_RECONNECT_TIME_MS = 1000;
```

Comment:
tiny nitpick: we're using _MILLIS to denote the unit in other constants, but _MS here.